### ISSUE TITLE
Cannot load up "curated" dataset

### ISSUE THREAD

**Issue by olivertam** [ergonyc/omicser#38]
I've been trying to set up a curated dataset on Omicser, but I have been encountering issues loading it
``` R
> omicser::run_app(options = list(launch.browser = TRUE))

Listening on http://127.0.0.1:6592
[1] " no database loaded... ."
Warning in file(con, "r") :
  cannot open file '/path/to/omicser_db_root/pbmc10k/additional_info.Rmd': No such file or directory
Warning: Error in file: cannot open the connection
  [No stack trace available]
[1] "db type - -omics"
```
It's unclear what the `additional_info.Rmd` file is, and also unclear why there is a connection issue (with no stack trace).
Any advice/help would be highly appreciated.

Thanks.

**Comment by olivertam** [comment]
I suspect that it's because the curated dataset is not "fully curated" in the form that is required for Omicser. However, given that there are steps in the curation (e.g. normalization) that I am trying to avoid (to preserve the original Cell Ranger output), it would be great to know what the minimal requirements are (files, configurations, fields) for a dataset to be visualizable on Omicser

Thanks.

**Comment by ergonyc** [comment]
This is resolved with  pr #41 . see also issue #39
